src/usr/local/www/diag_packet_capture.php

```diff
@@ -297,6 +296,11 @@ function hostport_array_fixer($items,$type) {
 		} elseif ($_POST['downloadbtn'] != "") {
 			//download file
 			send_user_download('file', $fp.$fn);
+		} elseif ($_POST['clearbtn'] != "") {
+			//clear log file
+			if (file_exists($fp.$fn)) {
```

Comment:
We have a function for this test+action, `unlink_if_exists($filename)`.

Reply:
I wanted to be conservative and use same code as the file.
As per your request, I modified the two occurrences, the exiting one and the one I added